build.gradle

```
@@ -17,10 +17,10 @@ dependencies {
 	//to change the versions see the gradle.properties file
 	minecraft "com.mojang:minecraft:${project.minecraft_version}"
 	mappings "net.fabricmc:yarn:${project.yarn_mappings}:v2"
-	modCompile "net.fabricmc:fabric-loader:${project.loader_version}"
+	modImplementation "net.fabricmc:fabric-loader:${project.loader_version}"
```

```suggestion
	implementation "net.fabricmc:fabric-loader:${project.loader_version}"
```
The loader is game agnostic and should not be remapped at all.

---

Yes and no. This will break loom doing this currenly, as loaders deps wont be pulled in

---

Then loom should be tweaked. Remapping loader is totally pointless.